tests/compiler/std/set.ts

```diff
@@ -1,3 +1,5 @@
+import { getValue } from "../named-import-default";
```

Comment:
Hmm, why we need this?

Reply:
Sorry, it's my carelessness. It auto imported by IED. I remove it.